pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -124,8 +124,18 @@ func (os *OpenStack) ListVolumes(limit int, startingToken string) ([]volumes.Vol
 // GetVolumesByName is a wrapper around ListVolumes that creates a Name filter to act as a GetByName
 // Returns a list of Volume references with the specified name
 func (os *OpenStack) GetVolumesByName(n string) ([]volumes.Volume, error) {
+	// Init a local thread safe copy of the Cinder ServiceClient
+	blockstorageClient, err := openstack.NewBlockStorageV3(os.blockstorage.ProviderClient, os.epOpts)
+	if err != nil {
+		return nil, err
+	}
+
+	// cinder online resize is available since 3.34 microversion
```

Comment:
we have a copy and paste error here. the microversion is not passed because of online resize but because filtering list volumes require the microversion

Reply:
This has been fixed.